addons/hw_screen/static/src/js/worker.js

```diff
@@ -0,0 +1,47 @@
+    $(function() {
+        var mergedHead = false;
+        var current_client_url = "";
+
+        function longpolling() {
+            $.ajax({
+                type: 'POST',
+                url: 'http://'+window.location.host+'/point_of_sale/get_serialized_order',
+                dataType: 'json',
+                beforeSend: function(xhr){xhr.setRequestHeader('Content-Type', 'application/json');},
+                data: JSON.stringify({jsonrpc: '2.0'}),
+
+                success: function(data) {
+                    var trimmed = $.trim(data.result.rendered_html);
+                    var parsedHTML = $('.shadow').html($.parseHTML(trimmed,keepScripts=true));
```

Comment:
I just realized keepScripts is not defined (this would blow up if you had used "use strict";). Why don't you simply call $.parseHTML(trimmed, true)? of course, with a comment to explain the danger of the second argument, for everyone who does not remember the exact API